### PR TITLE
Improve: remove nodename from taskinfo when RemoveTask

### PR DESCRIPTION
**Pull request by lowang-bh**
Signed-off-by: lowang_bh <lhui_wang@163.com>

### CODE REVIEW

File: pkg/scheduler/api/job_info.go [hunk 1]
```diff
@@ -116,8 +116,7 @@ func getTaskID(pod *v1.Pod) TaskID {
 
 // NewTaskInfo creates new taskInfo object for a Pod
 func NewTaskInfo(pod *v1.Pod) *TaskInfo {
-	req := GetPodResourceRequest(pod)
-	initResreq := GetPodResourceRequest(pod)
+	resReq := GetPodResourceRequest(pod)
```
Comment (k82cn) [inline pkg/scheduler/api/job_info.go:119]: I think we should distinguish initReq vs. req.

Comment (lowang-bh) [reply pkg/scheduler/api/job_info.go]: changed to 
```go
	initResReq := GetPodResourceRequest(pod)
	resReq := initResReq
```

**Comment by lowang-bh** [comment]
@tizhou86 GetDevicesIdleGPUMemory miss deviceID that has not been used ever, is it as design or not?

**Review by shinytang6 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**